src/chapters/typescript/compiling-typescript.rst

```diff
@@ -0,0 +1,64 @@
+Compiling TypeScript
+====================
+
+To work with TypeScript in our Node environment, we'll need to install the TypeScript
+module. In this section, you'll install the compiler and practice compiling and 
```

Comment:
Minor rephrasing: How about, "In this section, you will install the compiler and the practice compiling and running a small TypeScript program from the terminal."